app/scripts/controllers/permissions/background-api.test.js

```diff
@@ -34,22 +34,17 @@ describe('permission background API methods', () => {
       );
     });
 
-    it('throws if the specified account is already permitted', () => {
+    it('does not call permissionController.updateCaveat if the specified account is already permitted', () => {
```

Comment:
nit: can we update this phrase to indicate that it's testing the opposite of the above test? To me, it is a little cryptic understanding the intention of `permissionController.updateCaveat` in this test

```suggestion
    it('does not add a permitted account if the specified account is already permitted', () => {
```